apps/files/src/quickActions.js

```diff
@@ -53,8 +53,8 @@ function openNewCollaboratorsPanel(ctx) {
   })
 }
 
-function canShare(item) {
-  return item.canShare()
+function canShare(ctx) {
```

Comment:
If you want to have `store` and `item` as named variables, you can unwrap the ctx like this:
`function canShare({ item, store }) {`. Of course also fine like it is. 👍

Reply:
Right, 🤦 I did it already automatically after doing it in the handlers of quickActions. I got rid of the object completely and just pass it in.